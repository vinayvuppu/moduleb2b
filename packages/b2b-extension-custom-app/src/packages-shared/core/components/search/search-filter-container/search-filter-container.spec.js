import React from 'react';
import { shallow } from 'enzyme';
import SearchFilterContainer from '.';

describe('rendering', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(
      <SearchFilterContainer>{() => {}}</SearchFilterContainer>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
