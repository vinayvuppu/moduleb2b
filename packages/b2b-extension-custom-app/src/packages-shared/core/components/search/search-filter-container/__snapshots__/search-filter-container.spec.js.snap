// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering should match snapshot 1`] = `
<Shadow
  className="container"
  depth="1"
>
  <Collapsible
    isDefaultClosed={true}
  >
    <Component />
  </Collapsible>
</Shadow>
`;
